### PR TITLE
Robot definition improvements

### PR DESCRIPTION
**Pull request by VictorLamoine**
Follow up to https://github.com/ros-industrial/motoman_experimental/pull/44

- Rename meshes
- Use Collada smooth meshes for visual
-  Add test file
- Fix interface streaming file name
- Improve XACRO definition

# Visual
screenshot_20181119_110307

screenshot_20181119_110352

# Collisions
screenshot_20181119_110316

### CODE REVIEW

File: launch/load_mh24.launch [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" ?>
 <launch>
-	<param name="robot_description" command="$(find xacro)/xacro --inorder '$(find yaskawa_mh24_support)/urdf/mh24.xacro'" />
+  <param name="robot_description" command="$(find xacro)/xacro '$(find yaskawa_mh24_support)/urdf/mh24.xacro'" />
```
Comment (de-vri-es) [inline launch/load_mh24.launch:3]: Why remove `--inorder`? As far as I understood, that is the recommended use nowadays.

**Review by de-vri-es — COMMENTED** [review]
Could you elaborate a bit on the difference for the visual meshes? If they're indeed better I'll happily merge, but they are around 20% larger in size than the previous versions (atleast, the L axis mesh is).

File: launch/load_mh24.launch (hunk 1 above)
Comment (VictorLamoine) [reply launch/load_mh24.launch:3]: This is default starting from jade, and produces a warning starting from melodic.

**Comment by VictorLamoine** [comment]
The difference between the meshes are:
- Meshes are now shaded (https://answers.ros.org/question/226831/how-do-i-make-my-meshes-appear-shaded/), facets appear less visible
- Colors can be assigned per face (coloring the motor black for example). I've not done much coloring on the robot because it is at a remote place I don't go often. So right now the cables are colored gray and the connectors at the back are colored to. Coloring per face is not possible with STL files.

File: launch/load_mh24.launch (hunk 1 above)
Comment (de-vri-es) [reply launch/load_mh24.launch:3]: Ah I see. Leaving it out still produces a warning on kinetic/lunar I believe, but so be it. +1

**Comment by de-vri-es** [comment]
Alright, thanks for the clarifications and the PR. Merging :)

Like I mentioned in the other PR, if an ownership transfer to ROS-I for the repository is desired, let me know. Probably best to open an issue here in that case.

Thanks again!

**Pull request merged**